lib/bundle/commands/check.rb

```diff
@@ -52,5 +56,13 @@ def self.any_apps_to_install?
       current_apps = Bundle::MacAppStoreDumper.app_ids
       (requested_apps - current_apps).any?
     end
+
+    def self.any_formulae_to_start?
+      @dsl ||= Bundle::Dsl.new(Bundle.brewfile)
+      @dsl.entries.select { |e| e.type == :brew }.any? do |e|
+        formula = Bundle::BrewInstaller.new(e.name, e.options)
+        !((formula.restart_service? || formula.service_needs_started?) && Bundle::BrewServices.started?(e.name))
```

Comment:
The logic here is pretty hard to follow. Could you split `formula.restart_service? || formula.service_needs_started?` into a separate variable, `next` if it's `false`y and then have just `!Bundle::BrewServices.started?(e.name)` as the last statement? Thanks!